### PR TITLE
fix: svn checkout with can't convert string from 'UTF-8' to native encoding

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #1340 

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: build/resolver/svn/Dockerfile [hunk 1]
```diff
@@ -9,6 +9,13 @@ RUN apt-get update \
     && apt-get install -y subversion \
     && rm -rf /var/lib/apt/lists/*
 
+RUN apt-get clean \
```
Comment (supereagle) [inline build/resolver/svn/Dockerfile]: Why not merge this with the previous `RUN`?

Comment (zhujian7) [reply build/resolver/svn/Dockerfile]: Have fixed. PTAL

**Pull request merged**